src/referenceResolver.js

```diff
@@ -138,6 +138,7 @@ function unravelRange(range) {
 function formatArticle(article) {
   // remove dot and spaces + remove non digit trailing chars + replace unicode dash ‑ to standard -
   return article
+    .toUpperCase()
```

Comment:
I don't understand, using `.upperCase()` but having lowerCase in snapshot ?

Reply:
I've changed a test case to contain a lower case reference, and make sure it's resolved, that's why the snapshot has changed.

UpperCase() is used here to format the reference when we try to resolve it (in order to match the format in legi-data).

Reply:
thx !